lib/classes/Variables.test.js

```diff
@@ -76,6 +74,33 @@ describe('Variables', () => {
         expect(serverless.service.resources.bar).to.equal(barValue);
       });
     });
+
+    it('should delete null environment variables', () => {
+      const serverless = new Serverless();
+
+      serverless.service.provider.environment = {
+        SOME_VAR: 'some-value',
+        OTHER_VAR: null,
+      };
+      serverless.service.functions = {
+        hello: {
+          handler: 'hello.js',
+          environment: {
+            VAR_ONE: 'one',
+            VAR_TWO: null,
+          },
+        },
+      };
+
+      return serverless.variables.populateService().then(() => {
+        /* eslint-disable no-unused-expressions */
```

Comment:
Using `.to.be.undefined` instead of `.to.equal(undefined)` caused eslint to raise a `no-unused-expressions` error.   This fixes the failure.  Though, I'm not sure if it's the best way to handle it.

It might not be bad to disable that particular check on all `.test.js` files, since it's not unlikely to pop up elsewhere in tests as long as you're using Chai.